### ISSUE TITLE
Add a link to headline stats to appropriate visualisation on the page

### ISSUE THREAD

**Issue by taz-chiles** [open-innovations/yff-data#177]
Feedback from Chris referring to the headline stats: 
- Can they make it so if you click on the big number it goes to the relevant chart?

**Comment by gilesdring** [comment]
Will also need to work out how to show a particular tab in a tabbed visualisation

**Comment by slowe** [comment]
@gilesdring A few steps I've done in 4f0f9a5:
  - I have added `id` values to the `config` data for each of the visualisations in the main dashboard e.g. `src/dashboard/need/_data.yml` has had `id: headlines` added. The `src/_components/viz/main.njk` file then automatically adds that ID to the `pane` for that visualisation. There is no error checking so it is down to the person creating the page to avoid duplicate IDs.
  - In `src/_includes/partials/dashboard/headlines.njk` I have added the anchors to the links e.g. 
```
{% comp 'grid.block', { link: search.data('dashboard/labour-market/').url + '#headlines', width: '15rem' } %}
``` 

In the web page, `src/assets/js/tabbed.js` then does the following:
  - For each tab it moves the `id` attribute from the `pane` to the generated tab, keeps a note of which tab the `id` is connected to, and updates the `scroll-margin-top` CSS property to take account of the sticky header bar.
  - For each `pane` that still has an `id` (ones that aren't in tabs) it updates the `scroll-margin-top`.
  - For the `location.hash`, select any tab that has been connected to that `id`.
  - Updates `location.hash` again to force a scroll to the correct place.

**Comment by ODILMichelle** [comment]
Works for all visualisations (and jumps to correct tab) other than NEET line graph visualisation - where clicking through takes us to the top of the NEET page, rather than the visualisation on this page.

This may be intentional, but good to confirm if so.

**Comment by slowe** [comment]
@ODILMichelle To some extent it was intentional, yes. Partly because the issue only actually requests that the big headline numbers should go through to a relevant chart (as I'm not familiar with the content I only linked through to what seemed like a relevant dashboard). I only added anchor links for the three graphs to demonstrate getting to tabbed elements. If the NEET graph is required to have an anchor link I think that this be done by someone who is not me but who is likely to need to add these sorts of links in future. We should make sure that experience/knowledge isn't locked up in just me so it is useful for someone to see if the notes I wrote above are good enough/correct.
